app/scripts/services/authorization.js

```diff
@@ -96,16 +96,22 @@ angular.module("openshiftConsole")
       return _.get(cachedRulesByProject.get(projectName || currentProject), ['rules']);
     };
 
+    var checkRulesForResourceVerb = function(rules, resource, verb) {
+      return _.contains(rules[resource], verb) || _.contains(rules[resource], '*');
+    };
+
     var canI = function(verb, resource, projectName) {
```

Comment:
The `canI` filter directly takes resource in `<apiGroup>/<resource>` format and word 
https://github.com/openshift/origin-web-console/blob/master/app/views/browse/deployment-config.html#L50

The only place we need to distinguish between api group and a subresource is in the `canIAddToProject` but we are paasing there the original rules, not the normalized
https://github.com/openshift/origin-web-console/blob/master/app/scripts/services/authorization.js#L68